src/Sign.Core/DataFormatSigners/AzureSignToolSigner.cs

```diff
@@ -36,45 +36,55 @@ public AzureSignToolSigner(
             _logger = logger;
             _toolConfigurationProvider = toolConfigurationProvider;
 
-            // For PowerShell file extensions, see https://github.com/PowerShell/PowerShell/blob/2f4f585e7fe075f5c1669397ae738c554fa18391/src/System.Management.Automation/security/SecurityManager.cs#L97C1-L106C10
-            _supportedFileExtensions = new HashSet<string>(StringComparer.OrdinalIgnoreCase)
+            _signableFileTypes = new List<ISignableFileType>()
             {
-                ".app",
-                ".appx",
-                ".appxbundle",
-                ".cab",
-                ".cat",
-                ".cdxml",       // PowerShell cmdlet definition XML
-                ".dll",
-                ".eappx",
-                ".eappxbundle",
-                ".emsix",
-                ".emsixbundle",
-                ".exe",
-                ".msi",
-                ".msix",
-                ".msixbundle",
-                ".msm",
-                ".msp",
-                ".mst",
-                ".ocx",
-                ".ps1",         // PowerShell script files
-                ".ps1xml",      // PowerShell display configuration files
-                ".psd1",        // PowerShell data files
-                ".psm1",        // PowerShell module files
-                ".stl",
-                ".sys",
-                ".vbs",
-                ".vxd",
-                ".winmd"
+                // For PowerShell file extensions, see https://github.com/PowerShell/PowerShell/blob/2f4f585e7fe075f5c1669397ae738c554fa18391/src/System.Management.Automation/security/SecurityManager.cs#L97C1-L106C10
+                new SignableFileTypeByExtension(
+                    ".appx",
```

Comment:
`.app` removed here and added inside the `DynamicsBusinessCentralAppFileType` class.